api/services/ocean/cd/schemas/oceanCDWorkloadActiveOperations.yaml

```diff
@@ -1,19 +1,24 @@
 type: object
-required:
-  - workload
+title: Ocean CD Workload Active Operations
+description: Indicates the ongoing operations being applied to the workload
 properties:
   workloadId:
     type: integer
-    example: "405938231"
+    description: The ID of the workload
+    example: 405938231
   type:
     type: string
+    description: The type of the operation
+    example: retry
   operationStatus:
     type: string
+    description: The status of the operation
+    example: finished
   createdAt:
     type: string
-    format: date-time
-    example: "2018-11-05T12:55:50.000+0000"
+    description: The time when the operation was created
```

Comment:
why did you remove the format?

Reply:
I'm not sure the field is informative. The format isn't the same as displayed in this field. Also, as we have an example, we don't need it, or let's add smth like `yyyy-mm-ddTHH:mm:ss.fffZ`